Sources/Controllers/2.0/Settings/ImportExport/OAImportCompleteViewController.mm

```diff
@@ -325,6 +342,14 @@ -(void) loadCurrentRoutingMode
     [OARoutingHelper.sharedInstance setAppMode:currentRoutingMode];
 }
 
+- (NSString *)getTitleForSection
+{
+    NSString *headerText = OALocalizedString(@"import_complete_description");
+    if (_needRestart)
```

Comment:
Are you sure the restart is needed in ios?

Reply:
yes, some settings are not applied without restart